common/app/views/emailEmbed.scala.html

```diff
@@ -49,7 +49,15 @@
                 });
             }
 
-            revealPrivacyWordingWhenUserInteractsWithEmailField()
+            function informParentOfLoad(loadEvent) {
+                window.parent.postMessage({
+                    subject: 'emailEmbedPageLoaded',
+                    path: window.location.pathname,
+                }, window.origin);
+            }
+
+            window.addEventListener('DOMContentLoaded', informParentOfLoad);
```

Comment:
Remeber, functions are hoisted in JS, so you can register your listener higher up.